### ISSUE TITLE
Feedback on lessons app

### ISSUE THREAD

**Issue by Charlotte-Morgan** [kartoza/projecta#755]
This is looking really good. A few comments for tweaks and additions
is there some way to constrain the image size and proportions - Lessons viewed online on a wide screen have distorted images?
![image](https://user-images.githubusercontent.com/5743135/35955100-700899c0-0ce1-11e8-964f-a19a33cbdb35.png)
Introduce a minimum gap between the icons and headings - related to screen window size too
![image](https://user-images.githubusercontent.com/5743135/35955130-984b4c20-0ce1-11e8-8b62-6cf2cdc25b1a.png)

is there some way to enable a spell checker - if not all sheets will need to be proof read. This is regardless of distorted screen.

some strings appear to lack spaces between words:
![image](https://user-images.githubusercontent.com/5743135/35955077-5a33a838-0ce1-11e8-9500-6db9f7955992.png)

Add a field for last updated date and time - this will help to know if a translated versions are consistent - ie an english sheet may be updated but the translation not updated.
when I try to edit an existing sheet, i do not see all the fields - ie questions and answers - are there different user levels. I also do not see this when making a new sheet

**Comment by timlinux** [comment]
>is there some way to constrain the image size and proportions - Lessons viewed online on a wide screen have distorted images?

Yes I think we can constrain the image dimensions - we are also going to add some notes in the form to recommend the image dimensions. @Gustry or @myarjunar can you look at including a form widget which previews the image box in the letter box format?

> is there some way to enable a spell checker - if not all sheets will need to be proof read. This is regardless of distorted screen.

Not direectly easily, but depending on your browser you should be able to install something like grammarly (https://www.grammarly.com) which will add checking to your browser.

> Add a field for last updated date and time - this will help to know if a translated versions are consistent - ie an english sheet may be updated but the translation not updated.

OK good idea, easy to do.

> when I try to edit an existing sheet, i do not see all the fields - ie questions and answers - are there different user levels. I also do not see this when making a new sheet


No they are there you just need to mouse over the appropriate section and you will see widgets for them I tried to keep things clean looking when you are not directly editing content.

**Comment by Charlotte-Morgan** [comment]
Thanks @timlinux and @Gustry for the hints. My mouse over is not working - but @Gustry is going to look at why.

**Comment by Gustry** [comment]
About the mouse over, it's fixed and deployed @Charlotte-Morgan.

@christodewit I guess we would need to check for these missing spaces. Maybe it's the copy/paste from the ODT.
